content/Foundation/Summit/2023/home.pml

```diff
@@ -17,21 +17,22 @@ Minneapolis, USA
 <div class="summit_div">
 <div class="highlight">
 <h4></h4>
-<h4>Latest News: <a href="#summit-sponsors">Sponsors</a></h4>
+<h4>Latest News: <a href="#summit-abstracts">Videos, Slides and Abstracts</a></h4>
 </div>
 <p></p>
 </div>
 
 <div class="summit_div">
   <ul>
-    <li><a href="#summit-important-dates">Important Dates</a></li>
+    <!--li><a href="#summit-important-dates">Important Dates</a></li-->
     <li><a href="#summit-about">About</a></li>
     <!--li><a href="../principles/">Principles</a></li-->
-    <li><a href="#summit-registration">Registration</a></li>
+    <!--li><a href="#summit-registration">Registration</a></li-->
     <li><a href="#summit-location">Location and Venue</a></li>
     <li><a href="#summit-keynotes">Keynotes</a></li>
     <li><a href="#summit-panellists">Panellists</a></li>
-    <li><a href="#summit-program">Program</a></li>
+    <li><a href="program">Program</a></li>
```

Comment:
Change to "Program, Abstracts, Slides and Videos" and link to subsection below (and remove "Abstract" in menu).

And do the same in 2022 page